src/boot/fault_drawer.c

```diff
@@ -215,7 +215,7 @@ void FaultDrawer_FillScreen() {
     FaultDrawer_SetCursor(sFaultDrawerInstance->xStart, sFaultDrawerInstance->yStart);
 }
 
-void* FaultDrawer_FormatStringFunc(void* arg, const char* str, size_t count) {
+char* FaultDrawer_FormatStringFunc(char* arg, const char* str, size_t count) {
```

Comment:
What's the reasoning for this change?

Reply:
It was so it matches what ultralib's `outfun` which is what Printf takes as its function arg. However looking at the mdebug I think ultralib actually has it wrong and it should be changed. I'll revert this commit.